SIV_scraper

```diff
@@ -0,0 +1,215 @@
+setwd("L:/Prices/Dashboards/SIV")
+
+library(zoo)
```

Comment:
Are all these libraries necessary? Can they be reduced and can a standard approach be applied?